lib/omnibus/packagers/bff.rb

```diff
@@ -131,8 +131,14 @@ def write_scripts
     def write_gen_template
 
       # Get a list of all files
-      files = FileSyncer.glob("#{staging_dir}/**/*").map do |path|
-
+      files = FileSyncer.glob("#{staging_dir}/**/*").reject do |path|
+        # remove any files with spaces.
+        if path =~ /\s+/
```

Comment:
heh, i can't wait to see gems with checked in spec files that have unicode emojis in filenames and what those do to bff and solaris pkg packaging...